x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_log_expanded_tray.tsx

```diff
@@ -182,7 +182,7 @@ export const ActionsLogExpandedTray = memo<{
 }>(({ action, 'data-test-subj': dataTestSubj }) => {
   const getTestId = useTestIdGenerator(dataTestSubj);
 
-  const { startedAt, completedAt, command: _command, comment, parameters } = action;
+  const { hosts, agents, startedAt, completedAt, command: _command, comment, parameters } = action;
```

Comment:
Just a note, this `hosts` "name" is not the same as the one we've added in this PR under `EndpointActions.data` for automated actions from alerts.

Reply:
This is hosts from `ActionDetails` that could be created in two ways:
1. For Response Actions Results tab: it comes from `EndpointActions.data` 
2. For Normal Actions List it's calculated in `getActionDetailsList` 

Is this ok with you?

Reply:
That is right. But I don't see that change on the server side to handle that case for 1. That would go in `mapToNormalizedActionRequest` util function. And then on the list handler for action list you need to handle the `hosts` info being populated for each action based on whether it is automated or user initiated.

Reply:
We talked offline about this, and this is okay here. This hostname is indeed the one from normal actions and the change here to have consistent host names in the details.